submissions/omolayoOdupe/todo_app/src/components/TodoStatus.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const TodoStatus = (props) => {
+    return (
+        <div className='status'>
+            <span className='status1'>{props.itemsLeft().length}Items left</span>
```

Comment:
I think you can update the class names to what the content is about e.g `items-left`, `all-items`, `active-items` etc @OmolayoO